AppCenterCrashes/AppCenterCrashesTests/MSCrashesCategoryTests.m

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAppCenterInternal.h"
+#import "MSCrashesCategory.h"
+#import "MSCrashesPrivate.h"
+#import "MSMockUserDefaults.h"
+#import "MSTestFrameworks.h"
+#import "MSUtility+Application.h"
+
+#if TARGET_OS_OSX
+
+@interface NSApplication (CrashException)
+
+- (void)ms_reportException:(NSException *)exception;
+- (void)ms_sendEvent:(NSEvent *)theEvent;
+
+@end
+
+#endif
+
+static NSException *lastException;
+static void exceptionHandler(NSException *exception) { lastException = exception; }
+
+@interface MSCrashesCategoryTests : XCTestCase
+
+@end
+
+@implementation MSCrashesCategoryTests
+
+- (void)testActivateCategory {
+
+  // If
+#if TARGET_OS_OSX
```

Comment:
This should be for the entire method unless you want to verify something after `[MSCrashesCategory activateCategory]` in iOS/tvOS.